doc/using_statistics.md

```diff
@@ -134,22 +136,93 @@ Server: Jetty(9.1.5.v20140505)
 }
 ```
 
+**Connecting The Focus To The Publisher For PubSub Method**
+
+Upon start of the Jitsi Videobridge, it will send the following stanza to the focus
```

Comment:
Ah, our component and focus are kind of jumbled together. I'll update the wording to be more clear. Thank you for the feedback.